subsys/bluetooth/host/Kconfig

```diff
@@ -351,6 +351,24 @@ config BT_OOB_DATA_FIXED
 	  value. This option should only be enabled for debugging and should
 	  never be used in production.
 
+config BT_KEY_DISPLACEMENT
```

Comment:
maybe `ROTATION` or `CIRCULAR_BUFFER`?

Reply:
> maybe ROTATION or CIRCULAR_BUFFER?

Yeah, I also like those better than the current proposal. Also see my suggestion. I think I'll eventually favour whichever is the shortest yet adequately describes the policy :)

Reply:
Does name "ROTATION" explain that old keys are dropped well enough?

Reply:
`BT_KEYS_OVERWRITE_OLDEST` is fine and descriptive